isolate.sh

```diff
@@ -181,8 +181,10 @@ function prereq() {
     cs_operator_found=false
 
     while read -r ns; do
-        cs_version=$("${OC}" get csv -n "${ns}" | grep common-service-operator | awk '{print $7}')
-        if [[ -n "${cs_version}" ]]; then
+        cs_version=$("${OC}" get csv -n "${ns}" | grep common-service-operator | awk '{print $7}' || echo "")
+        if [[ $cs_version == ""]]; then
+            error "Error happened when trying to get ibm-common-service-operator csv"
```

Comment:
@qpdpQ 
Could you update the error message to something like this?
```suggestion
            error "Failed to get ibm-common-service-operator csv in namespace ${ns}."
```